src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+$array: (1 'mon', 2 'tue', 3 'wed', 4 'thu', 5 'fri', 6 'sat', 7 'sun');
```

Move this variable to the separate variables file
```suggestion
$days-week: (1 'mon', 2 'tue', 3 'wed', 4 'thu', 5 'fri', 6 'sat', 7 'sun');
```

---

corrected